src/platform/Darwin/ConfigurationManagerImpl.cpp

```diff
@@ -66,10 +66,7 @@ bool ConfigurationManagerImpl::_CanFactoryReset()
     return true;
 }
 
-void ConfigurationManagerImpl::_InitiateFactoryReset()
-{
-    PlatformMgr().ScheduleWork(DoFactoryReset);
-}
+void ConfigurationManagerImpl::_InitiateFactoryReset() {}
```

Comment:
should we log that factory reset not yet implemented or should we add a comment on why this is empty?

Reply:
I will add a log. Seems a good idea to at least see some outputs if it is called.